app/assets/javascripts/initializers/initScrolling.js

```diff
@@ -212,6 +212,7 @@ function insertArticles(articles) {
   }
   articles.forEach(function insertAnArticle(article) {
     var existingEl = document.getElementById('article-link-' + article.id);
+    var currentUserId = userData().id;
```

Comment:
I'm passing the `currentUserId` into `buildHTMLArticle` to perform the check that hides the Bookmark button.
I kept with the file's convention of `var` instead of `const`.